nomad/operator_endpoint_test.go

```diff
@@ -125,7 +125,9 @@ func TestOperator_RaftGetConfiguration_ACL(t *testing.T) {
 
 func TestOperator_RaftRemovePeerByAddress(t *testing.T) {
 	t.Parallel()
-	s1 := TestServer(t, nil)
+	s1 := TestServer(t, func(c *Config) {
+		c.RaftConfig.ProtocolVersion = raft.ProtocolVersion(2)
```

Comment:
Remove the equivalent raft=v3 changes from other tests in this file as they should work with the default configuration.